src/components/GoodCard/GoodCard.jsx

```diff
@@ -0,0 +1,5 @@
+export const GoodCard = ({ good }) => (
+  <ul>
+    <li data-cy="Good">{good}</li>
+  </ul>
+);
```

Comment:
This component should be converted to TypeScript. Define the prop types using TypeScript interfaces or types to ensure type safety.